package.json

```diff
@@ -26,7 +26,8 @@
                 ],
                 "filenamePatterns": [
                     "suite*.rc",
-                    "suite*.rc.*"
+                    "suite*.rc.*",
+                    "*.cylc"
```

Comment:
Just a thought, would adding global.rc, flow.rc, flow-tests.rc make sense? I have loaded my files of these, renaming them to suite.rc and the extension works well?

Reply:
We could make the change to `flow.cylc` pretty soon so this'll do for now.